plugins/chat/assets/javascripts/discourse/components/chat/routes/channel-info-members.gjs

```diff
@@ -125,76 +125,78 @@ export default class ChatRouteChannelInfoMembers extends Component {
   }
 
   <template>
-    {{#if this.site.mobileView}}
-      <LinkTo
-        class="c-back-button"
-        @route="chat.channel.info.settings"
-        @model={{@channel}}
-      >
-        {{icon "chevron-left"}}
-        {{i18n "chat.members_view.back_to_settings"}}
-      </LinkTo>
-    {{/if}}
-    {{#if this.showAddMembers}}
-      <MessageCreator
-        @mode={{this.addMembersMode}}
-        @channel={{@channel}}
-        @onClose={{this.hideAddMember}}
-        @onCancel={{this.hideAddMember}}
-      />
-    {{else}}
-      <div class="c-channel-members">
-        <DcFilterInput
-          {{autoFocus}}
-          @filterAction={{this.mutFilter}}
-          @icons={{hash right="search"}}
-          @containerClass="c-channel-members__filter"
-          placeholder={{this.filterPlaceholder}}
+    <div class="c-routes --channel-info-members">
```

Comment:
I wonder if we should we keep the modifier naming the same here as drawer (ie. --members), I realise the file name is slightly different too so likely the cause of this.

Reply:
Yes Thats a good idea :+1: